### PR TITLE
TASK: Adjust version numbers in tooling configuration

### PR DESCRIPTION
**Pull request by bwaidelich**
Updates for the 7.0 branch:

* conf.py of neos/flow
* .travis.yml
* apigen.yml

### CODE REVIEW

File: Neos.Flow/Documentation/conf.py [hunk 1]
```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'dev-master'
+version = '7.0'
```
Comment (albe) [inline Neos.Flow/Documentation/conf.py:62]:
```suggestion
version = 'dev-7.0'
```
Was that `dev-7.0` intended? Because it should be just 7.0, no?

**Review by albe — APPROVED** [review]

File: Neos.Flow/Documentation/conf.py (hunk 1 above)
Comment (bwaidelich) [reply Neos.Flow/Documentation/conf.py:62]: Well, I just copied what @kdambekalns did in 37396baf4725379f7d24146738fe17f1b8a30d4f

Comment (kdambekalns) [reply Neos.Flow/Documentation/conf.py:62]: Rightly so.

Comment (kdambekalns) [reply Neos.Flow/Documentation/conf.py:62]: Ah, that targets the next changed line, actually…

File: Neos.Flow/Documentation/conf.py [hunk 2]
```diff
@@ -59,9 +59,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = 'dev-master'
+release = 'dev-7.0.x'
```
Comment (kdambekalns) [inline Neos.Flow/Documentation/conf.py:64]:
```suggestion
release = '7.0.0'
```

Yes, that never get's updated during the lifecycle (yet), but still…

Comment (kdambekalns) [reply Neos.Flow/Documentation/conf.py:64]: Hm… actually that's what I did in https://github.com/neos/flow-development-collection/commit/37396baf4725379f7d24146738fe17f1b8a30d4f. 🙈

Probably _because_ it is not updated automagically for releases and rendered from the branch tip anyway. All is well!

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**